stubs/pyinstaller/METADATA.toml

```diff
@@ -0,0 +1,6 @@
+version = "5.3.*"
+requires = ["types-setuptools"]
+
+[tool.stubtest]
+# Most modules are not meant to be used, yet are not marked as private
+ignore_missing_stub = true
```

Comment:
Or should I add said modules to `stubtest_allowlist.txt` ?

Reply:
Definitely — though, it might be worth making the scale of the problem clear in the comment (138!!)

Reply:
I just realized I can use wildcards in the allowlist. Doing so I was able to bring it just under 40 items. Is this more acceptable or should I keep `ignore_missing_stub = true` ? 
```
# fake module, only exists once the app is frozen
pyi_splash
# undocumented and clearly not meant to be exposed
PyInstaller.__main__.generate_parser
PyInstaller.__main__.run_build
PyInstaller.__main__.run_makespec
PyInstaller.utils.hooks.conda.lib_dir
# A mix of modules meant to be private, and shallow incomplete type references for other modules
PyInstaller.building.*
PyInstaller.depend.analysis.*
# Most modules are not meant to be used, yet are not marked as private
PyInstaller.isolated._parent.*
PyInstaller.archive.*
PyInstaller.config
PyInstaller.configure
PyInstaller.depend.bindepend
PyInstaller.depend.bytecode
PyInstaller.depend.dylib
PyInstaller.depend.imphook
PyInstaller.depend.utils
PyInstaller.exceptions
PyInstaller.hooks.*
PyInstaller.lib.*
PyInstaller.loader.*
PyInstaller.log
PyInstaller.utils.cliutils.*
PyInstaller.utils.conftest
PyInstaller.utils.git
PyInstaller.utils.hooks.django
PyInstaller.utils.hooks.gi
PyInstaller.utils.hooks.qt
PyInstaller.utils.hooks.subproc.*
PyInstaller.utils.hooks.tcl_tk
PyInstaller.utils.misc
PyInstaller.utils.osx
PyInstaller.utils.run_tests
PyInstaller.utils.tests
PyInstaller.utils.win32.*
```

Reply:
Ah! Let's go with that then, please :)

If possible, it's definitely best not to have to enable `--ignore-missing-stub`. With stubtest on maximum strictness, our CI starts to fail when new methods are added to classes (etc.), so it's a very helpful way of ensuring that our stubs don't get too out of date.